scripts/data_importer_service/data_import_service.py

```diff
@@ -63,9 +66,18 @@ def process_data_file(self, path, file_extension):
                 if isUnsupportedValue:
                     assessment[variable] = None
 
+        var_set = set(assessment_variables)
+        assessment_variables = list(var_set)
+        assessment_variables.sort()
+
         self.data_file.update(
-            {"metadata": metadata, "participant_assessments": participant_assessments}
+            {
+                "metadata": metadata,
+                "participant_assessments": participant_assessments,
+                "assessment_variables": assessment_variables,
+            }
         )
```

Comment:
is this going to create the new collections, or this is what gets passed to the DPDash app and then we'll store them?

Reply:
nvm I see in the description that this gets sent to DPDash 🙌